public/app/features/explore/RichHistory/RichHistoryCard.test.tsx

```diff
@@ -100,6 +100,33 @@ describe('RichHistoryCard', () => {
       expect(wrapper.find({ title: 'Add comment' })).toHaveLength(1);
       expect(wrapper.find({ title: 'Edit comment' })).toHaveLength(0);
     });
+    it('should open update comment form when edit comment button clicked', () => {
+      const wrapper = setup({ query: starredQueryWithComment });
+      const editCommentButton = wrapper.find({ title: 'Edit comment' });
+      editCommentButton.simulate('click');
+      expect(wrapper.find({ 'aria-label': 'Update comment form' })).toHaveLength(1);
+    });
+    it('should close update comment form when escape key pressed', () => {
+      const wrapper = setup({ query: starredQueryWithComment });
+      const editCommentButton = wrapper.find({ title: 'Edit comment' });
+      editCommentButton.simulate('click');
+      wrapper.simulate('keydown', { key: 'Escape' });
+      expect(wrapper.find({ 'aria-label': 'Update comment form' })).toHaveLength(0);
+    });
+    it('should close update comment form when enter and shift keys pressed', () => {
```

Comment:
As we allow to update comment with enter key and either shift or ctrl, can we have an additional test scenario checking ctrl+enter key case?